tempy/tempy.py

```diff
@@ -402,6 +406,51 @@ def wrap(self, other):
         other.append(self)
         return self
 
+    def wrap_many(self, *args, strict=False):
+        """Wraps different copies of this element inside all empty tags
+        listed in params or param's (non-empty) iterators.
+
+        Returns list of copies of this element wrapped inside args
+        or None if not succeeded, in the same order and same structure,
+        i.e. args = (Div(), (Div())) -> value = (A(...), (A(...)))
+
+        If on some args it must raise TagError, it will only if strict is True,
+        otherwise it will do nothing with them and return Nones on their positions"""
+        wcopies = []
+        failure = []
+
+        def wrap_next(tag, idx):
+            nonlocal wcopies, failure
+            next_copy = self.__copy__()
+            try:
+                return next_copy.wrap(tag)
+            except TagError:
+                failure.append(idx)
+                return None
+
+        arg_idx = -1
+        for arg in args:
```

Comment:
using `arg_idx` that way is very java-ish, in python is better to use `enumerate`:

```
for args_idx, arg in enumerate(args):
    if isinstance(arg, DOMElement):
        . . .
```
Same few lines below.